### PR TITLE
Change Cifar10's model optimizer

### PR DESCRIPTION
**Pull request by arthurPignet**
The RMSprop optimizer can be changed to adam with a different learning rate. It greatly improves the training performance, as shown below. 

image
image
image

However, even if I tried a few learning rates and decay values, these are propably not the perfect hyperparameters, do not hesitate to suggest another setup. 

Signed-off-by: arthurPignet <arthur.pignet@mines-paristech.fr>

### CODE REVIEW

File: mplc/dataset.py [hunk 1]
```diff
@@ -189,8 +189,8 @@ def generate_new_model(self):
         model.add(Dense(self.num_classes))
         model.add(Activation('softmax'))
 
-        # initiate RMSprop optimizer
-        opt = RMSprop(learning_rate=0.0001, decay=1e-6)
+        # initiate Adam optimizer
+        opt = Adam(learning_rate=0.001, decay=1e-5)
```
Comment (RomainGoussault) [inline mplc/dataset.py:193]: https://keras.io/api/optimizers/adam/
what is decay ? it is the beta_1 in the doc ?

File: mplc/dataset.py [hunk 2]
```diff
@@ -189,8 +189,8 @@ def generate_new_model(self):
...
 
         # Let's train the model using RMSprop
```
Comment (RomainGoussault) [inline mplc/dataset.py:195]: to remove

**Review by RomainGoussault — APPROVED** [review]

**Review by Thomas-Galtier — APPROVED** [review]

File: mplc/dataset.py (hunk 2 above)
Comment (arthurPignet) [reply mplc/dataset.py]: 👍

File: mplc/dataset.py (hunk 1 above)
Comment (arthurPignet) [reply mplc/dataset.py:193]: No, beta_1 and beta_2 are for the momentum. 
As far as I understood the doc, the decay is for the learning rate.
